### DISCUSSION TITLE
Notificações do WhatsApp os

### DISCUSSION THREAD

**Discussion by andremp15** [RamonSilva20/mapos#2388] [Ideas]
Bom dia a todos, tive essa ideia em relações as notificações de os pelo WhatsApp, expressei minha ideia nessa imagem
 abaixo.![IMG-20240419-WA0001.jpg](https://github.com/RamonSilva20/mapos/assets/117945999/781bcfe5-60bc-488d-af3a-2be046f8e7f9)

**Comment by gleissonneves** [comment]
A ideia é interessante, seria importante abrir como uma issue

**Reply by andremp15** [reply]
Boa noite amigo eu faço esse procedimento manualmente, um bloco de nota, já que não tem como fazer no mapa e eu não sou programador, fica desta forma 

*UCM INFORMÁTICA*. Assistência Técnica Especializada*💻🖥️📱🕹️ 

*Numero da OS nº 3293*  *Estator Orçamento*  *Data 26/04/2024* 

*Olá, Lucas Morais Assunção, Tudo bem?*😀. Sou o técnico responsável, *André*, Temos novidades, do seo *Notebook Acer Aspire E1-572-BR691*, segue a baixo as informações. 

*Defeito relatado pelo cliente:* 
Cliente reclama de travamento e lentidão. Teclado com defeito. 

*Laudo técnico relatado pelo técnico:* 
Foi contratado inseto dentro do Notebook, precisa fazer a limpeza preventiva, teclado em curto sendo necessário a substituição,  lentidão que o cliente mencionou é devido HD mecânico está com defeito de bad block recomendado trocar por SSD 

*Observação*
Entregue sem carregador. 

Limpeza preventiva --------- *R$ 150.00*
SSD 240gb ---------------------- *R$ 275.00*
Memória DDR4 4gb--------- *R$ 135.00*
Teclado --------------------------- *R$ 170.00*
Formatação -------------------- *R$ 150.00* 

Sub Total ---------------------------- *R$ 840.00*
Desconto ---------------------------- *R$.  75.00*
Total ----------------------------------- *R$ 765.00* 

Parcelamos em 3x sem juros, 💳 visa ou Mastercard. 😀
                   
Se tiver qualquer duvida e só nos constatar.
